### ISSUE TITLE
1.10 sponge forge java exception - no data from satellite

### ISSUE THREAD

**Issue by fusionmc-evilscoop** [zmaster587/AdvancedRocketry#975]
Minecraft Version:
1.10
Forge Version:
forge-1.10.2-12.18.3.2477
Advanced Rocketry Version:
AdvancedRocketry-1.10.2-1.2.5a.jar
LibVulpes Version:
LibVulpes-1.10.2-0.2.6-universal.jar

Sponge version:
spongeforge-1.10.2-2477-5.2.0-BETA-2731.jar

#Please Pastebin crash
---------------------
Crash Report:
https://pastebin.com/raw/ArYzRqtL
Explain issue in detail:

A player on the sponge forge server launched a satellite, but is getting no data from it.
Found the above errors in the logs

**Comment by fusionmc-evilscoop** [comment]
ok updated libvulpes and rocketry, and now no errors, but still no data...
Is there some forge permission nodes that need setting ?

AdvancedRocketry-1.10.2-1.2.5a-57
LibVulpes-1.10.2-0.2.6-2-universal

**Comment by fusionmc-evilscoop** [comment]
Ok tried again, looks like rocket is not actually going into orbit, it just sits at top of altitude bar

**Comment by fusionmc-evilscoop** [comment]
and this happens on satellite destroy - 
https://pastebin.com/raw/wYsWFNwP

My theory is the sat, once out of range is being removed from the server, and not "going" were its meant to, thus it sits at the top of alt, with 0 vel, and half a tank of fuel, as this was last data received.
Since it cant get to were it needs to, the terminal cannot connect

**Comment by fusionmc-evilscoop** [comment]
Bunch of errors from trying with a station this time:-
https://pastebin.com/raw/sR92YX1A

**Comment by fusionmc-evilscoop** [comment]
attempting a manned rocket...
rocket wont evan take off...
getting out of rocket, and removing the chip (programmed with dst and luna selected)
Client crashed with the following -
https://pastebin.com/raw/0mccfV6a

Am i missing some worlds ?

**Comment by fusionmc-evilscoop** [comment]
updated to latest versions
spongeforge-1.10.2-2477-5.2.0-BETA-2782.jar
LibVulpes-1.10.2-0.2.7-5-universal.jar
AdvancedRocketry-1.10.2-1.2.5a-64.jar
forge-1.10.2-12.18.3.2511-universal.jar

Still not working......

**Comment by fusionmc-evilscoop** [comment]
guess this is not getting looked at...as the mod is unusable on my server, it will be removed next update....

Shame I had high hopes for this mod.....

**Comment by zmaster587** [comment]
Have you tried running it without sponge?  Sponge tends to mess with things particularly preventing rockets from reaching alt ( see https://github.com/zmaster587/AdvancedRocketry/issues/999#issuecomment-353676495 ).  And Im sorry I cannot look at everything all the time.  I'm one guy who already works a full time job and keeps house and I literally get more issues a day than I have time to fix even if I stopped sleeping.

This https://pastebin.com/raw/0mccfV6a should have been fixed in a semi recent build.
 
This only indicates that somebody tried to drop a satellite chip as an item into the world https://pastebin.com/raw/ArYzRqtL
